lib/camp.js

```diff
@@ -628,13 +635,14 @@ function respondWithFile(req, res, path, ifNoFile) {
     }
     res.mime(p.extname(realpath).slice(1));
 
-    // Cache management.
-    if (+(new Date(req.headers['if-modified-since'])) >= +stats.mtime) {
+    // Cache management (compare timestamps with second-level precision).
+    var since = req.headers['if-modified-since'];
+    if (since && (Math.abs(stats.mtime - new Date(since)) < 1000)) {
       res.statusCode = 304;
       res.end();  // not modified.
       return;
     }
-    res.setHeader('Last-Modified', stats.mtime.toGMTString());
+    res.setHeader('Last-Modified', stats.mtime.toUTCString());
```

Comment:
It gives exactly the same output currently, but moves one step further away from deprecation. A future step we could take is to `.replace(/GMT/g, 'UTC')`, but this is [not spec-compliant](https://www.w3.org/Protocols/rfc2616/rfc2616-sec3.html#sec3.3.1) yet, and I haven't actually tested it on browsers.